test-build/main.css

```diff
@@ -0,0 +1,4 @@
+div {
```

Comment:
I don't see this file being used anywhere, should it be ignored from the repo?

Reply:
It shouldn't be there at all. I'm not sure what created it. Going to assume it was my mistake and remove